python/ui.py

```diff
@@ -229,12 +234,29 @@ def update_tree(new_tree):
             changed = True
         elif key in ("KEY_BACKSPACE", "\b", "\x7f"):
             update_tree(tree.merge(highlighted))
+        elif key == "v" :
+            if tree.annotation(highlighted) == "vectorize":
+                update_tree(tree.annotate(highlighted, ""))
+            else:    
+                update_tree(tree.annotate(highlighted, "vectorize"))
+
+        elif key == "u" :
+            if tree.annotation(highlighted) == "unroll":
+                update_tree(tree.annotate(highlighted, ""))    
+            else:
+                update_tree(tree.annotate(highlighted, "unroll"))
+            
         elif key == "\n":
             key = "ENTER"
             drag = None if drag else loop_version(tree, highlighted)
         render(changed)
-        if key == "u":
+        if key == "U":
             trees = trees[:-1]
+
+    print("Printing actions file\n")
```

Comment:
if you think we need print statements here, maybe we should consider using a Python-standard debug printing lib?

Reply:
One more file that got added by mistake... Sorry.